src/compile.directive.js

```diff
@@ -0,0 +1,28 @@
+// Copied from https://docs.angularjs.org/api/ng/service/$compile
+angular.module('ml.common', [], function($compileProvider) {
```

Comment:
Please remove the second argument here. See https://docs.angularjs.org/guide/module#creation-versus-retrieval for details.